CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+    2. [Submitting a Pull Request](#pullRequest)
+3. [Development Setup](#developmentSetup)
+4. [Branching & Naming Conventions](#branchesAndNames)
+5. [Commit Message Guidelines](#commits)
+6. [Style & Linting](#styleAndLinting)
+7. [Testing](#testing)
+8. [Documentation](#documentation)
+9. [Release Process (maintainers only)](#release)
+10. [License](#license)
+
+## Code of Conduct <a name="codeOfConduct"></a> 
+
+We adopt the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/). By participating in this project you agree to abide by its terms. Please read it before interacting with the community.
+
+## How to Contribute <a name="howToContribute"></a>
+
+### Reporting Bugs / Suggesting Enhancements <a name="bugAndEnhancements"></a>
+
+1. Search first – check the existing issues to avoid duplicates.
+2. Open a new issue; use the provided template if you want to add a new term.
+3. Provide a clear title.
+4. Add a concise description that includes:
+    - What you expected to happen.
+    - What actually happened (include error messages, stack traces, screenshots, etc.).
+    - Steps to reproduce (if applicable).
+    - Your environment (OS, Node/Python version, browser, etc.).
+
+### Submitting a Pull Request (PR) <a name="pullRequest"></a>
+
+#### 1. Setup up the Development Environment (see Development Setup below).
+#### 2. Fetch from original repository.
+`git fetch upstream`
+
+#### 3. Merge all changes from fork and you local repo
+`git merge upstream/master`
+
+#### 4. Create a branch (see naming conventions below).
+create a branch based on master state and goes there
+`git checkout -b ISS123-meaningful-name master`
+
+#### 5. Make your changes; keep them focused on a single purpose.
+
+#### 6. Make sure you changes only affect the intended files and lines
+`git diff`
+
+#### 7. Test you changes on PaNET.csv with ROBOT
+???
+
+#### 8. Update documentation (md files in the repository and documentation webpage)
+#### 9. Commit using the guidelines below and push your branch to your fork.
+```
+git add PaNET.csv
+git commit
+git push origin HEAD
+```
+
+#### 10. Open a PR against the master branch.
+#### 11. Fill out the PR template, linking any related issue (e.g., Fixes #123, Contributes to #420).
+
+
+> Tip: If you’re unsure whether a change is welcome, open a draft PR or comment on the related issue first.
+
+## Development Setup <a name="developmentSetup"></a>
+
+### 1. Clone the repo (necessary only once)
+```
+git clone https://github.com/ExPaNDS-eu/ExPaNDS-experimental-techniques-ontology
+cd your-repo
+```
+
+### 2. Add remote repository (necessary only once)
+`git remote add upstream git@github.com:ExPaNDS-eu/ExPaNDS-experimental-techniques-ontology.git`
+
+### 3. Install ROBOT (necessary only once)
+The PaNET.owl is generated from an Excel/CSV file using [ROBOT](https://central.sonatype.com/artifact/org.obolibrary.robot/robot?smo=true) software. Here are some instructions to generate the OWL file:
+???
```

Comment:
We perhaps should also mention the `panet-build` container image that provides a complete environment for building PaNET.  This is likely an easy way for someone to get started.